server/server.go

```diff
@@ -115,13 +117,15 @@ func (h *SlackHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// First check if path matches our BasePath
-	// If yes then attempt to decode it to match on Command or CallbackID
+	// If yes then attempt to decode it to match on Command or CallbackID / InteractionType
 	// If no then match custom paths
 	if r.URL.Path == h.basePath {
-		// Attempt to parse as a slash command first
-		// Ignore errors here as the library always returns a non-nil struct
-		sc, _ := slack.SlashCommandParse(r)
-		if len(sc.Command) > 0 {
+		if err := r.ParseForm(); err != nil {
+			fmt.Errorf("Unable to parse request from Slack: %s", err)
```

Comment:
What do you mean to do with the error you have just created? Log it? Return to sender as a 500?